src/python/pants/option/global_options.py

```diff
@@ -1611,6 +1611,7 @@ def compute_pantsd_invalidation_globs(
             (
                 "!*.pyc",
                 "!__pycache__/",
+                ".gitignore",
```

Comment:
This is technically a user-facing bugfix rather than internal, but I'm fine skipping the separate CI burn.

Reply:
Agreed. I think we should rename the PR to be "Include `.gitignore` in default `pantsd_invalidation_globs`", so that it goes into "bug fix" for the changelog.

We can include the typo fix and not include in the PR title